packages/flutter_tools/lib/src/dart/analysis.dart

```diff
@@ -46,6 +46,7 @@ class AnalysisDriver {
   DriverOptions options;
 
   String get sdkDir {
+    // ignore: deprecated_member_use
```

Comment:
The deprecated call is taken care of w/ https://github.com/flutter/flutter/commit/bde8a5f5a68c763af52780a4c033f37b9fa99620, but the question remains whether we should get rid of `options.dartSdkPath`.  Looking at `analyze_once.dart` it does appear we support a 'dart-sdk' flag so removing it would be a breaking change...